core/io/close_spec.rb

```diff
@@ -45,6 +45,26 @@
       @io.close.should be_nil
     end
   end
+
+  ruby_version_is '2.5.0' do
+    it 'raises an IOError when the stream is closed in another thread' do
```

Comment:
raises an IOError *with a clear message*

Reply:
done. I have updated the PR.